bower.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "adapt-contrib-pageLevelProgress",
-    "version": "2.0.8",
-    "framework": "^2.0.0",
+    "version": "2.1.0",
```

Comment:
Would suggest a patch-level bump instead like the other releases, unless there are any new features.